Pinkesh/migrate remaining components (#96)
FlutterFlow/flutterflow-documentation: docs/resources/control-flow/user-interactivity/forms/form-widgets/checkbox.md

---
slug: /resources/forms/checkbox

> title: Checkboxes

Reviewer: Just Checkbox

Author: Done!

tags: [Widgets, Form Elements]
keywords: [FlutterFlow, Checkbox, CheckboxGroup, CheckboxListTile, Multiple Choice]
description: Learn how to add Checkbox, CheckboxGroup, and CheckboxListTile widget in your FlutterFlow app.
sidebar_position: 3
---

> # Checkboxes

Reviewer: Just Checkbox

Reviewer: To differentiate just say Checkbox Widgets here on the title and then Checkbox later as h2

Author: Done!

> In FlutterFlow, checkboxes are versatile input widgets used to capture binary choices from users, such as true/false or yes/no options. They are ideal for situations where you need to present users with options that can be individually selected or deselected. FlutterFlow provides three primary checkbox widgets: [**Checkbox**](#checkbox), [**CheckboxListTile**](#checkboxlisttile), and [**CheckboxGroup**](#CheckboxGroup). Each of these widgets offers distinct features and use cases, making it easy to tailor your app's interface to your specific needs.

Reviewer: ```suggestion
In FlutterFlow, a checkbox is a versatile input widget used to capture binary choices from users, such as true/false or yes/no options. It is ideal for situations where you need to present users with options that can be individually selected or deselected. FlutterFlow provides three primary variations of the checkbox widget: [**Checkbox**](#checkbox), [**CheckboxListTile**](#checkboxlisttile), and [**CheckboxGroup**](#CheckboxGroup). Each of these widgets offers distinct features and use cases, making it easy to tailor your app's interface to your specific needs.
```

Author: Updated!

            left: 0,
            width: '100%',
            height: '100%',
            colorScheme: 'light'
        }}
        frameborder="0"
        loading="lazy"
        webkitAllowFullScreen
        mozAllowFullScreen
        allowFullScreen
        allow="clipboard-write">
    </iframe>
</div>
<p></p>

## CheckboxListTile

The **CheckboxListTile** widget combines the functionality of a checkbox with a [ListTile](../../../../ui/widgets/built-in-widgets/list-grid.md#listtile-widget), providing a more comprehensive option for displaying checkboxes alongside additional information. Unlike the Checkbox this widget includes a title, and an optional subtitle, all within a single, cohesive element.

CheckboxListTile is ideal for use cases where you want to provide more context or descriptive text alongside the checkbox, such as in a settings menu or a form with detailed options.

## CheckboxGroup

The **CheckboxGroup** widget allows you to present a group of checkboxes as a single entity. This is particularly useful when you want users to select multiple options from a list. Each checkbox within the group can be checked or unchecked independently of the others.

> :::tip[Widget State]

Reviewer: This should be added to the top of page as all the widgets here have a widget state. Add them before talking about the variations in details.

Author: Thanks for catching!
